tensorly/solvers/nnls.py

```diff
@@ -15,7 +15,7 @@ def hals_nnls(
     epsilon=0.0,
     callback=None,
 ):
-    """
+    r"""
```

Comment:
If we make this into a raw-string, then we also need to remove all the double backslashes in the docstring. Alternatively, we can set all single backslashes into double backslashes and skip the r.

Also, it's probably better to use an uppercase R instead of a lowercase one since the lowercase R will trigger regex code highlighting in some editors.

Reply:
Thanks for comments, it would be better to remove the 'r'. 😄